cmd/buildah/from.go

```diff
@@ -69,7 +69,12 @@ func init() {
 	flags.StringVar(&opts.creds, "creds", "", "use `[username[:password]]` for accessing the registry")
 	flags.StringVarP(&opts.format, "format", "f", defaultFormat(), "`format` of the image manifest and metadata")
 	flags.StringVar(&opts.name, "name", "", "`name` for the working container")
-	flags.StringVar(&opts.pull, "pull", "true", "pull images from the registry if newer or not present in store, if false, only pull images if not present, if always, pull images even if the named images are present in store, if never, only use images present in store if available")
+	flags.StringVar(&opts.pull, "pull", "missing", `pull images from the registry values:
+always: pull images even if the named images are present in store,
+missing: pull images if the named images are not present in store,
+never: only use images present in store if available,
+newer: pull if the image on the registry is newer than the in store`)
```

Comment:
Possibly this should be `ifnewer` and not `newer`? [Related PR that fixes the docs](https://github.com/containers/buildah/pull/5524).

Reply:
I would rather go with Podman's definitions so these matchup.

Reply:
The description of "newer" looks like it's missing a word between "the" and "in"